virtual_labs/infrastructure/kc/auth.py

```diff
@@ -44,16 +44,9 @@ def verify_jwt(
 
     try:
         token = header.credentials
-        decoded_token = kc_auth.decode_token(
-            token=token,
-            key=get_public_key(),
-            options={
-                "verify_aut": True,
-                "verify_signature": True,
-                "verify_exp": True,
-            },
-        )
-    except Exception:
+        decoded_token = kc_auth.decode_token(token=token, validate=True)
```

Comment:
I've tested a few tokens now. An invalid token throws an error so `validate=True` seems to verify the token correctly.